thirdparty/ofqf/qoscserver.cpp

```diff
@@ -136,12 +136,14 @@ void QOscServer::readyRead()
               
               		//qDebug() << " after transformation to OSC-RegExp path is" << path;
               
-              		QRegExp exp( path );
-              		foreach( PathObject* obj, paths ) {
-              		      if ( exp.exactMatch( obj->_path ) )
-              				        obj->signalData( arguments );
-              		      }
-              		}
+                  QRegExp exp(path);
+                  for (PathObject* obj : paths) {
```

Comment:
Please keep as much as possible from the orginal source, including the formatting